pkg/models/repository.go

```diff
@@ -69,11 +89,14 @@ func (in *Repository) DeepCopyInto(out *Repository) {
 
 func (r *Repository) MapForUpdate() map[string]interface{} {
 	forUpdate := make(map[string]interface{})
-	forUpdate["LastReadTime"] = r.LastReadTime
-	forUpdate["LastReadError"] = r.LastReadError
 	forUpdate["URL"] = r.URL
 	forUpdate["Public"] = r.Public
```

Comment:
This is the line I commented in above comment, and it avoided to update the column to false when the repository is introspected. But it will avoid a repository could be updated from public <> private once it was created.